02-syntax.Rmd

```diff
@@ -207,6 +205,17 @@ do_something_very_complicated("that", requires, many, arguments,
                               )
 ```
 
+As described under [Argument names], you can omit the argument names
+for very common arguments (i.e. for arguments that are used in almost every 
+invocation of the function). Unnamed arguments also go on the same line as the
```

Comment:
Maybe just "can also" - i.e. you don't _have_ to do it.

Or maybe "short unnamed arguments"

Reply:
Ok, so I changed to 
> Short unnamed arguments can also go on the same line as the function name.